test/sagemaker_tests/tensorflow/tensorflow2_training/requirements.txt

```diff
@@ -12,7 +12,7 @@ pytest-rerunfailures==9.0
 pytest-xdist
 requests==2.24
 mock
-sagemaker<2
+sagemaker<3
```

Comment:
nit: Consider changing this to `sagemaker>=2,<3` to be more specific for versions.

Also, were the changes to `tensorflow1_training` and `inference` reverted?

Reply:
inference was not in this PR - tf1 test changes were reverted due to failures and will be followed up in a separate PR, please see the description.